robot-name/robot-name.spec.js

```diff
@@ -60,4 +60,33 @@ describe('Robot', () => {
     expect(modifyInternal).toThrow();
   });
 
+
+  xit('should test for sequential names', () => {
+    const name1 = robot.name;
+    robot.reset();
+    const name2 = robot.name;
+    robot.reset();
+    const name3 = robot.name;
+
+    expect(areSequential(name1, name2)).not.toBe(true);
```

Comment:
You can expect falsy values in different ways in Jasmine.

```
expect(areSequential(name1, name2)).not.toBe(true);
expect(areSequential(name1, name2)).toBe(false);
expect(areSequential(name1, name2)).toBeFalsy();
```

I'm not very fan of the first one. What do you think? What do you think is the most readable?